pg/pg_test.go

```diff
@@ -0,0 +1,39 @@
+package pg
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aria-afk/go-get-tickets/utils"
+)
+
+func setup() {
+	os.WriteFile("pg_test.env", []byte("PG_CONN_STRING=postgresql://postgres:test@localhost/template1"), 0755)
```

Comment:
will this write the file in the root directory? might be nice to put it in a test directory that we can gitignore or something

Reply:
They will get added to the modules dir (so in this case `pg/`) also, oops for all the file stuff we can just do `defer ...` to ensure nothing gets left around